### DISCUSSION TITLE
I'm back!

### DISCUSSION THREAD

**Discussion by MatusGuy** [Mt-roblox/Mt#4] [Blog]
I'm back at working on the Mt module after a few months!

I will start by improving object implementation, because, right now, it's very messy, as it clones the members of the parent classes and probably wastes a lot of memory.

**Comment by MatusGuy** [comment]
Before that, I'm making [a tool](https://github.com/MatusGuy/files2inst) to boost development. Sort of like Rojo's DOM. Allows you to convert an rbxmx file to a bunch of json files and vice-versa.

**Comment by HedTB** [comment]
Asked

**Reply by MatusGuy** [reply]
brur
